clientgui/ViewWork.cpp

```diff
@@ -745,6 +766,19 @@ wxString CViewWork::OnListGetItemText(long item, long column) const {
             case COLUMN_STATUS:
                 strBuffer = work->m_strStatus;
                 break;
+            case COLUMN_ESTIMATEDCOMPLETION:
+                if (work->m_fCPUTime > 0) {
+                    wxDateTime now = wxDateTime::Now();
+                    wxTimeSpan time_to_completion = convert_to_timespan(work->m_fTimeToCompletion);
+                    wxDateTime estimated_completion = now.Add(time_to_completion);
+                    FormatDateTime(estimated_completion.GetTicks(), strBuffer);
+                } else {
+                    strBuffer = FormatTime(0);
```

Comment:
What if the task was started but then paused? It still will show increased time each second I believe.

Reply:
What does BoincTasks say in that scenario? I think it would be helpful if both tools followed the same rules.

Reply:
@AenBleidd wouldn't that be the expected behavior? If I pause a task, I would personally expect the Estimated Completion to continue to increase until I un-pause the task.

@RichardHaselgrove I'm not sure as I haven't used BoincTasks before. But, I can install it and take a look.

Reply:
The column name is 'Estimated Completion' but you can't estimate it for paused task. It doesn't matter whether user on scheduler paused the task, if you will just show some datetime in future that is increased every second - it means no sense because it says actually nothing about the real estimated completion time. If you want, you cat try to examine the scheduler code, find its queue and using this information try to predict when every task will start and then make the final calculation for estimated completion. But it's complex, maybe even too complex. So for the tasks that are paused or not started (let's say, the tasks that are not currently running) I'd just show empty field or dash symbol within this column. But it's up to you which path to use

Reply:
OK, thanks for the feedback! That makes sense. And I agree, examining the scheduler code and using that to make a calculation for estimated completion is probably overkill for this purpose. I'll just go ahead and show the same 3 dashes (---) when a task is not currently running.